### ISSUE TITLE
False triggers in sunlight

### ISSUE THREAD

**Issue by bill-orange** [adafruit/Adafruit-VL53L0X-ToF-Distance-Sensor-PCB#5]
I am using the VL53LOX as a 'car position detector' in my garage.  If nothing is in range of the sensor an LED is off.  If the car is 30" to 40" from the rear wall the LED is on solid.  Closer than 30" and the LED blinks.  Each state times out in 15 sec. to an LED off state.  Loop has a 60 ms delay at the end.

All is well when the garage door is closed.  When it is open the LED flickers as random IR hits the sensor.

By hardware or software, is there a way to eliminate this annoyance?

**Comment by kemalcan2** [comment]
Hello, bill-orange

Did you solve this problem? 

Best Regards.

**Comment by bill-orange** [comment]
No. Friends have speculated that the sensor is triggering on dust in the air.  This is possible.  Reflected light from the street could be another cause.  Either in hardware or software there needs to be better anti-aliasing, whatever the cause.

**Comment by kemalcan2** [comment]
Okay Bill, thank you for your answer. 

Take care.

**Comment by marciogsc** [comment]
On the ST forum it says that in sunlight the sensor works up to 50 cm.

[https://community.st.com/s/question/0D50X00009XkYI8SAN/vl53l0x-ambient-light-sensitivity](url)
